ffda-gluon-usteer/README.md

```diff
@@ -69,3 +78,47 @@ Clients require to maintain `band_steering.min_snr` for `band_steering.interval`
 ## Site configuration
 
 The site configuration provides the site-wide default configuration for usteer.
+
+## Example
+
+To fully leverage the capabilities of the `ffda-gluon-usteer` package in your Gluon firmware, you can add the following usteer configuration block to your `site.conf` file:
+
+    usteer = {
+      max_signal_diff = 10,
+      min_signal = -75,
+      roam_trigger = -70,
+      signal_avg_weight = 0.5,
+      load_balancing = true,
+    }
```

Comment:
aren't those options only part of the `client_steering` ?
The `client_steering` options are currently not noted in the configuration section. Would be good to add this there as well

I would prefer to also have this as a highlighted code block for consistency?

Reply:
I didn't check it. I just took the first part of it from Dortmund.
And the second part I found somewhere. 

My suggestion is just a stub as a base for sophisticated example 

Feel free to edit it

Reply:
I didn't check it. I just took the first part of it from Dortmund.
And the second part I found somewhere. 

My suggestion is just a stub as a base for sophisticated example 

Feel free to edit it